management-console/src/screens/StickVisualization/StickVisualization.tsx

```diff
@@ -1,36 +1,67 @@
 import { CircularProgress, Typography } from "@mui/material";
-import { useGetStickPosistion } from "../../api/stick"
+import { StickEvent, useGetStickEvents } from "../../api/stick"
+import { useCallback, useState } from "react";
 
-const getPositionBackground = (currentPosition: string = "", position: string) => position.toString() === currentPosition ? "#96e996" : "lightgrey"
+const getPositionBackground = (currentPosition: number, position: number) => position === currentPosition ? "#96e996" : "lightgrey"
 
 export const StickVisualization = () => {
-  const { stickPosition, connected } = useGetStickPosistion();
+  const [stickLog, setStickLog] = useState<string[]>([]);
+  const [cellSequence, setCellSequence] = useState<number[]>([]);
+  const [stickPosition, setStickPosition] = useState(5)
+  const stickEventHandler = useCallback((msg: StickEvent) => {
+    if (msg.type === "cellChange") {
+      const newCell = parseInt(msg.value)
+      setStickPosition(newCell)
+      setCellSequence(prev => [...prev, newCell])
+    }
+    else if (msg.type === "wordEnded") {
+      setStickLog(prev => [`Word Ended with cells ${msg.value}`, ...prev])
```

Comment:
Word ended is not a clear phrase for me
What about something like "word recorded", "word received"
I am not sure about this yet